compiler/AST/type.cpp

```diff
@@ -276,6 +277,10 @@ const char* toString(Type* type, bool decorateAllClasses) {
       // de-sugar chpl__c_void_ptr, which is used internally and is a distinct
       // type from c_ptr(void)
       retval = "raw_c_void_ptr";
+    } else if (vt == dtStringC) {
+      // present dtStringC type as c_ptrConst(c_char) instead of the internal
+      // name chpl_c_string or c_string_rehook.
+      retval = "c_string";
```

Comment:
Seems like this doesn't do anything, as it is written. The comment seems to imply it would need `retval = "chpl_c_string";`.

Reply:
the comment was wrong, but without this we print `chpl_c_string` if you ask to print the `type` of a `c_string` variable. 
```chapel
var x1 : c_string;
writeln(x1.type:string); // prints chpl_c_string
```

Reply:
OK, in that case fixing the comment will suffice

Reply:
I fixed up this comment to indicate that the presented name is `c_string` here